### PR TITLE
add 7.4 in basic features

### PR DESCRIPTION
**Pull request by hfxsd**
### What is changed, added or deleted? (Required)

Add 7.4 in basic features

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: basic-features.md [hunk 1]
```diff
@@ -20,231 +20,231 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | E | E | N | N | N | N | N | N | N | N | N |
-| [TiFlash 存算分离架构与 S3 支持](/tiflash/tiflash-disaggregated-and-s3.md) | E | E | E | N | N | N | N | N | N | N | N |
+| 管理，可视化和工具 | 7.4 | 7.3 | 7.2 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [TiDB Dashboard 图形化展示](/dashboard/dashboard-intro.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [TiDB Dashboard 持续性能分析功能](/dashboard/continuous-profiling.md) | Y | Y | Y | Y | Y | Y | E | E | N | N | N | N |
+| [TiDB Dashboard Top SQL 功能](/dashboard/top-sql.md) | Y | Y | Y | Y | Y | Y | E | N | N | N | N | N |
+| [TiDB Dashboard SQL 诊断功能](/information-schema/information-schema-sql-diagnostics.md) | Y | Y | Y | Y | Y | E | E | E | E | E | E | E |
+| [TiDB Dashboard 集群诊断功能](/dashboard/dashboard-diagnostics-access.md) | Y | Y | Y | Y | Y | E | E | E | E | E | E | E |
+| [Grafana 中的 TiKV-FastTune 面板](/grafana-tikv-dashboard.md#tikv-fasttune-面板) | Y | E | E | E | E | E | E | E | E | E | E | E |
```
Comment (hfxsd) [inline basic-features.md:224]:
```suggestion
| [Grafana 中的 TiKV-FastTune 面板](/grafana-tikv-dashboard.md#tikv-fasttune-面板) | E | E | E | E | E | E | E | E | E | E | E | E |
```

File: basic-features.md [hunk 2]
```diff
@@ -20,231 +20,231 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
-| [TiFlash Pipeline 执行模型](/tiflash/tiflash-pipeline-model.md) | E | E | N | N | N | N | N | N | N | N | N |
-| [TiFlash 副本选择策略](/system-variables.md#tiflash_replica_read-从-v730-版本开始引入) | Y | N | N | N | N | N | N | N | N | N | N |
-| [索引合并](/explain-index-merge.md) | Y | Y | Y | Y | Y | Y | E | E | E | E | E |
-| [基于 SQL 的数据放置规则](/placement-rules-in-sql.md) | Y | Y | Y | Y | Y | E | E | N | N | N | N |
-| [Cascades Planner](/system-variables.md#tidb_enable_cascades_planner) | E | E | E | E | E | E | E | E | E | E | E |
-| [Runtime Filter](/runtime-filter.md) | Y | N | N | N | N | N | N | N | N | N | N |
+| 高级 SQL 功能 | 7.4 | 7.3 | 7.2 | 7.1 | 6.5 | 6.1 | 5.4 | 5.3 | 5.2 | 5.1 | 5.0 | 4.0 |
+|---|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+| [Prepare 语句执行计划缓存](/sql-prepared-plan-cache.md) | Y | Y | Y | Y | Y | Y | Y | Y | E | E | E | E |
+| [非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md) | Y | E | E | E | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:95]: v7.4 GA

File: basic-features.md [hunk 3]
```diff
@@ -20,231 +20,231 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [执行计划管理 (SPM)](/sql-plan-management.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [根据历史执行计划创建绑定](/sql-plan-management.md#根据历史执行计划创建绑定) | Y | Y | Y | Y | E | N | N | N | N | N | N | N |
+| [下推计算结果缓存 (Coprocessor Cache)](/coprocessor-cache.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | E |
+| [Stale Read](/stale-read.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | N | N |
+| [Follower Read](/follower-read.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [通过系统变量 `tidb_snapshot` 读取历史数据](/read-historical-data.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [Optimizer hints](/optimizer-hints.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [MPP 执行引擎](/explain-mpp.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | N |
+| [MPP 执行引擎 - compression exchange](/explain-mpp.md#mpp-version-和-exchange-数据压缩) | Y | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [TiFlash Pipeline 执行模型](/tiflash/tiflash-pipeline-model.md) | Y | E | E | N | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:105]: v7.4 GA

File: basic-features.md [hunk 4]
```diff
@@ -20,231 +20,231 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [前台限流](/tikv-configuration-file.md#前台限流) | Y | Y | Y | Y | Y | E | N | N | N | N | N | N |
+| [后台限流](/tikv-configuration-file.md#后台限流) | E | E | E | E | E | N | N | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [PITR](/br/br-pitr-guide.md) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E| E | E | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | E | E | E | N |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | E | E | E | N | N | N | N | N | N | N | N | N |
+| [TiFlash 存算分离架构与 S3 支持](/tiflash/tiflash-disaggregated-and-s3.md) | Y | E | E | E | N | N | N | N | N | N | N | N |
```
Comment (hfxsd) [inline basic-features.md:247]: v7.4 GA @qiancai

File: basic-features.md [hunk 5]
```diff
@@ -20,231 +20,231 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [DM WebUI](/dm/dm-webui-guide.md) | E | E | E | E | E | E | N | N | N | N | N | N |
+| [前台限流](/tikv-configuration-file.md#前台限流) | Y | Y | Y | Y | Y | E | N | N | N | N | N | N |
+| [后台限流](/tikv-configuration-file.md#后台限流) | E | E | E | E | E | N | N | N | N | N | N | N |
+| [基于 EBS 的备份和恢复](https://docs.pingcap.com/zh/tidb-in-kubernetes/v1.4/volume-snapshot-backup-restore) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [PITR](/br/br-pitr-guide.md) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [全局内存控制](/configure-memory-usage.md#如何配置-tidb-server-实例使用内存的阈值) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [RawKV 跨集群复制](/tikv-configuration-file.md#api-version-从-v610-版本开始引入) | E| E | E | E | E | N | N | N | N | N | N | N |
+| [Green GC](/system-variables.md#tidb_gc_scan_lock_mode-从-v50-版本开始引入) | E | E | E | E | E | E | E | E | E | E | E | N |
+| [资源管控 (Resource Control)](/tidb-resource-control.md) | Y | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | E | E | E | N | N | N | N | N | N | N | N | N |
```
Comment (Oreoxmt) [inline basic-features.md:246]:
```suggestion
| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | E | E | E | N | N | N | N | N | N | N | N | N |
| [自动管理后台任务](/tidb-resource-control.md#管理后台任务) | E | N | N | N | N | N | N | N | N | N | N | N |
```
@songrijie PTAL

Comment (songrijie) [reply basic-features.md:246]:
```suggestion
| [Runaway Queries 自动管理](/tidb-resource-control.md#管理资源消耗超出预期的查询-runaway-queries) | E | E | E | N | N | N | N | N | N | N | N | N |
| [后台任务资源管控](/tidb-resource-control.md#管理后台任务) | E | N | N | N | N | N | N | N | N | N | N | N |
```

File: basic-features.md (hunk 4 above)
Comment (hfxsd) [inline basic-features.md:247]:
```suggestion
| [TiFlash 存算分离架构与 S3 支持](/tiflash/tiflash-disaggregated-and-s3.md) | Y | E | E | E | N | N | N | N | N | N | N | N |
| [TiDB Service scope](/system-variables.md#tidb_service_scope-从-v740-版本开始引入) | E | N | N | N | N | N | N | N | N | N | N | N |


```

**Review by Frank945946 — APPROVED** [review]

File: basic-features.md [hunk 6]
```diff
@@ -20,231 +20,231 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [Key 分区](/partitioned-table.md#key-分区) | Y | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [List 分区](/partitioned-table.md#list-分区) | Y | Y | Y | Y | Y | Y | E | E | E | E | E | N |
+| [List COLUMNS 分区](/partitioned-table.md#list-columns-分区) | Y | Y | Y | Y | Y | Y | E | E | E | E | E | N |
+| [List 和 List COLUMNS 分区表的默认分区](/partitioned-table.md#默认的-list-分区) | Y | Y | N | N | N | N | N | N | N | N | N | N |
+| [`EXCHANGE PARTITION`](/partitioned-table.md) | Y | Y | Y | Y | Y | E | E | E | E | E | E | N |
+| [`REORGANIZE PARTITION`](/partitioned-table.md#重组分区) | Y | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [`COALESCE PARTITION`](/partitioned-table.md#减少分区数量) | Y | Y | Y | Y | N | N | N | N | N | N | N | N |
+| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | Y | Y | Y | E | E | E | E | N | N |
+| [Range COLUMNS 分区](/partitioned-table.md#range-columns-分区) | Y | Y | Y | Y | Y | N | N | N | N | N | N | N |
+| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | Y | Y | Y | Y | E | N | N | N | N | N | N | N |
```
Comment (joey-yez) [inline basic-features.md:160]:
```suggestion
| [Range INTERVAL 分区](/partitioned-table.md#range-interval-分区) | Y | Y | Y | Y | E | N | N | N | N | N | N | N |
| [分区表转换为非分区表](/partitioned-table.md#将分区表转换为非分区表) | Y | N | N | N | N | N | N | N | N | N | N | N |
| [对现有表进行分区](/partitioned-table.md#对现有表进行分区) | Y | N | N | N | N | N | N | N | N | N | N | N |
```

**Review by qiancai — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: basic-features.md [hunk 7]
```diff
@@ -20,231 +20,235 @@ aliases: ['/docs-cn/dev/basic-features/','/docs-cn/dev/experimental-features-4.0
...
+| [直方图](/statistics.md) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| 扩展统计信息（多列） | E | E | E | E | E | E | E | E | E | E | E | N |
+| 统计反馈 | N | N | N | N | N | 已废弃 | 已废弃 | E | E | E | E | E |
+| [统计信息自动更新](/statistics.md#自动更新) | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y | Y |
+| [快速分析](/system-variables.md#tidb_enable_fast_analyze) | E | E | E | E | E | E | E | E | E | E | E | E |
+| [动态裁剪](/partitioned-table.md#动态裁剪模式) | Y | Y | Y | Y | Y | Y | E | E | E | E | N | N |
+| [收集部分列的统计信息](/statistics.md#收集部分列的统计信息) | E | E | E | E | E | E | E | N | N | N | N | N |
+| [限制统计信息的内存使用量](/statistics.md#统计信息收集的内存限制) | E | E | E | E | E | E | N | N | N | N | N | N |
+| [随机采样约 10000 行数据来快速构建统计信息](/system-variables.md#tidb_enable_fast_analyze) | E | E | E | E | E | E | E | E | E | E | E | E |
+| [锁定统计信息](/statistics.md#锁定统计信息) | E | E | E | E | E | N | N | N | N | N | N | N |
```
Comment (ran-huang) [inline basic-features.md:178]:
```suggestion
| [锁定统计信息](/statistics.md#锁定统计信息) | Y | E | E | E | E | N | N | N | N | N | N | N |
```

**Pull request merged**